plugin/testdata/externalPluginSchema.json

```diff
@@ -1160,9 +1160,7 @@
     "aws.ec2InstanceMetadataJSON": {
         "label": "metadata.json",
         "singleton": true,
-        "methods": [
-            "read"
-        ],
+        "methods": [],
```

Comment:
This ensures that there's a test case to check that empty method arrays in entry schema JSONs are also supported.